### PR TITLE
Url input v02 with api jyothsna

### PR DESCRIPTION
**Pull request by Jyothsna-GC**
Updated with text describing app 
Introduced Sentimentalists logo

### CODE REVIEW

**Comment by abirdonwire** [comment]
Hi Jyothsna, I've just asked Jamie to review as well but I can't merge this pull request as there are lots of conflicts existing files that aren't part of the URLInput component. The logo and main areas are in the master branch if you want to take a look or want me to talk you through it.

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1012/142647.527:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (jsjohnstone) [inline debug.log]: This log file doesn't need to be committed - can we delete it?

File: public/index.html [hunk 2]
```diff
@@ -2,7 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="/SentimentalistsTN.png" />
```
Comment (jsjohnstone) [inline public/index.html:5]: What's the purpose of the `%PUBLIC_URL` tag in the original favicon reference? Should we be including it? I think it helps ensure the location of the 'public/' dir is correct.

File: src/App.js [hunk 3]
```diff
@@ -1,15 +1,72 @@
...
+      <div className={classes.containerStyle}>
+    {!request
+    ?
+      <URLInput getURL={ getURL } setRequest={ setRequest }/>
+    :
+      <div>
+      <DisplayResults displayResults={JSON.stringify(request.results)} />
+      <h3>{request.url}</h3>
+      <h3>{JSON.stringify(request.results)}</h3>
+      <h3>{request.error}</h3>
```
Comment (jsjohnstone) [inline src/App.js:58]: Can't remember if error handling is part of our original scope, but maybe there should be an `if` statement here to show either the results or the error? I don't believe the backend will ever return both (it'll either return an error object or a results object).

Comment (jsjohnstone) [reply src/App.js:58]: Oh, realise now that these are debug strings I put in - the `<DisplayResults>` component is actually doing the 'results'... the question still stands of whether we should put a switch between results or error state.

File: src/components/URLInput/URLInput.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+function URLInput(props) {
+  const [text, setText] = useState("");
+  const classes = useStyles();
+
+  const handleTextChange = (event) => {
+    setText(event.target.value);
+  };
+
+  const handleClick = () => {
+    props.getURL(text);
```
Comment (jsjohnstone) [inline src/components/URLInput/URLInput.js:40]: @abirdonwire Should the `getURL` function exist within the URLInput component as opposed to in App? Or does it need to exist at all? It looks like we 'set' the text state, but then don't use it when it comes to handling a click (instead we use the getURL function - but could just use the 'text' variable)?

File: src/components/URLInput/URLInput.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        if (!response.ok) {
+          // get error message from body or default to response status
+          const error = (data && data.message) || response.status;
+          return Promise.reject(error);
+        }
+
+        props.setRequest(JSON.parse(data));
+      })
+      .catch((error) => {
+        console.error("There was an error!", error);
```
Comment (jsjohnstone) [inline src/components/URLInput/URLInput.js:61]: We should handle errors better - but maybe that's a V1 issue?

File: src/components/URLInput/URLInputStyles.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+const useStyles = makeStyles({
+    aBorderStyle : {
+        border: '3px solid red'
+    }
+});
+
+export default function Hook(){
+    const classes = useStyles();
+    return <Button className={classes.aBorderStyle}>Boobs</Button>
```
Comment (jsjohnstone) [inline src/components/URLInput/URLInputStyles.js]: What does this do 😂

**Review by jsjohnstone — COMMENTED** [review]
A couple of comments - they can either be resolved/kept as issues to deal with later, or dealt with.. happy to discuss/chat about any of them 👍

File: src/components/URLInput/URLInputStyles.js (hunk 6 above)
Comment (abirdonwire) [reply src/components/URLInput/URLInputStyles.js]: Hahahhaha! This does absolutely nothing! It's from when I was playing around with CSS styles and it's been branched. Definitely a deletion required.

File: debug.log (hunk 1 above)
Comment (abirdonwire) [reply debug.log]: I think we can.

File: public/index.html (hunk 2 above)
Comment (abirdonwire) [reply public/index.html:5]: I don't know what this %PUBLIC_URL%/ is, it was just initialised as standard. I'm happy to delete it, I don't think it's important.
This favicon that's been added though, will conflict as I've already pushed our logo favicon to master.

File: src/App.js (hunk 3 above)
Comment (abirdonwire) [reply src/App.js:58]: I think this is what Ana was looking at in some form today and I definitely agree we should have something that does just this.

File: src/components/URLInput/URLInput.js (hunk 4 above)
Comment (abirdonwire) [reply src/components/URLInput/URLInput.js:40]: Want to have a look with me tomorrow? I did that function initially before we had the API connected so if it's defunct now we have setState then that's fine.

File: src/components/URLInput/URLInput.js (hunk 5 above)
Comment (abirdonwire) [reply src/components/URLInput/URLInput.js:61]: Definitley needs addressing though. It's likely going to be a question or test we'll end up having.

**Comment by abirdonwire** [comment]
There's still a lot of work to be done here, files are definitely not ready to be merged with master. Would you all be happy for me to close this?

File: src/App.js (hunk 3 above)
Comment (jsjohnstone) [reply src/App.js:58]: This is planned in the v1.5: https://github.com/TheSentimentalists/SentimentalistsApp-Frontend/issues/6

File: src/App.js [hunk 7]
```diff
@@ -1,40 +1,75 @@
...
-          
-          <Grid container className={classes.footer}>
-            <Grid item xs={12}>
-              <h5>The Sentimentalists &copy; 2020</h5>
-            </Grid>
-          </Grid>
-        </div>
-      </Container>
+
+    <div>
```
Comment (abirdonwire) [inline src/App.js:41]: @GCJyo Can we have a chat about this bit? Just need to clarify a couple of things before we approve the pull request.

**Pull request closed without merging**